testonly/hammer/hammer.go

```diff
@@ -165,8 +166,26 @@ func (c MapConfig) String() string {
 		c.MapID, c.EPBias, c.Operations, c.EmitInterval, c.RetryErrors)
 }
 
+// SetULimit calls `ulimit -n` to set the maximum number of open files.
+// See https://wilsonmar.github.io/maximum-limits/
```

Comment:
Given this is basically a syscall wrapper, it might be nicer to include a [canonical](https://www.unix.com/man-page/posix/3posix/setrlimit/) [reference](http://man7.org/linux/man-pages/man2/setrlimit.2.html), rather than a random webpage (which might disappear tomorrow).

Reply:
Thanks